app/src/main/java/org/gdg/frisbee/android/event/EventOverviewFragment.java

```diff
@@ -272,12 +277,15 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()){
-            case R.id.add_calendar: addEventToCalendar();return true;
-            case R.id.navigate_to: launchNavigation();return true;
-            case R.id.view_event_url: if (mEvent.getEventUrl() != null) {
+            case R.id.add_calendar: 
+                addEventToCalendar();
+                return true;
+            case R.id.navigate_to: 
+                launchNavigation();
+                return true;
+            case R.id.view_event_url: 
                 launchUrl(mEvent.getEventUrl());
```

Comment:
I would still keep the check for valid event url to make it apparent that this might go wrong and add nullable to getEventUrl()